src/apps/ethereum/tokens.py

```diff
@@ -1,12 +1,12 @@
 def token_by_chain_address(chain_id, address):
-    if not address:
-        return None
     for token in tokens:
         if chain_id == token[0] and address == token[1]:
             return token
     return None
 
 
+UNKNOWN_TOKEN = True
```

Comment:
@jpochyla I wans't sure to what value I should declare this :-/ . Also I've forgotten `const()` haven't I?

Reply:
I suggest `UNKNOWN_TOKEN = object()`